POSTGRESQL_SETUP_GUIDE.md

```diff
@@ -0,0 +1,14 @@
+# Setting up PostgreSQL .
+
+## Installing PostgreSQL 10
+Open any of the links below and follow the installation instructions.
+- [Windows ](https://www.postgresql.org/download/windows/)
+- [Mac](https://www.postgresql.org/download/macosx/)
+- [Linux](https://www.postgresql.org/download/linux/)
+- [Heroku](https://devcenter.heroku.com/articles/heroku-postgresql)
+
+##  Set up Seqeulize ORM
+- Run  `npm install` to install [sequelize](https://www.npmjs.com/package/sequelize) and [sequelize-cli](https://www.npmjs.com/package/sequelize-cli)
+- Update the `env` variable `PG_USERNAME` and `PG_PASSWORD`(Your Postgres username and password) 
+- From the terminal, navigate to the `app/db/` 
+- Run `node_modules/.bin/sequelize init:migration`
```

Comment:
Can substitute with `npx sequelize init:migration`.

Reply:
If we use `npx`, is there any reason to install / declare the `sequelize-cli` package as a dependency?

Reply:
Possibly. I believe in this context npx will run the local sequelize-cli dependency in your node_modules, basically a cleaner version of what you have here (it obviates the need to adjust for relative paths by adding ../../, for instance). This means you can control the version and don't have to wait for npx to install it. But you could also do `npx sequelize-cli` if you wanted to eliminate the dependency.

Also looks like you want `npx sequelize db:migrate` to run the migrations. I think init:migration(s) does something different.

Reply:
Running `db:create` first will create the database if it doesn't exist, but not set it up. Then I think you'd run `db:migrate`.

Reply:
The Sequelize docs use the no-local-install npx sequelize-cli approach FWIW.

Reply:
Super helpful comments. Thanks @chriszs!